### PR TITLE
config: enable merge and use table namespace classifier in default

### PR DESCRIPTION
**Pull request by Connor1996**

### CODE REVIEW

File: server/config.go [hunk 1]
```diff
@@ -385,7 +385,7 @@ const (
 	defaultMaxReplicas          = 3
 	defaultMaxSnapshotCount     = 3
 	defaultMaxPendingPeerCount  = 16
-	defaultMaxMergeRegionSize   = 0
+	defaultMaxMergeRegionSize   = 20
```
Comment (siddontang) [inline server/config.go]: 20 byte?

Comment (disksing) [reply server/config.go]: 20MB.

Comment (siddontang) [reply server/config.go]: can you make it more specifical?

Comment (Connor1996) [reply server/config.go]: @siddontang what do you mean? declare unit explicitly

Comment (siddontang) [reply server/config.go]: use 20 * 1024 * 1024

or rename to RegionSizeMb exjplictily. 

IMO, size is for byte, not mega byte.

Comment (Connor1996) [reply server/config.go]: I prefer renaming to RegionSizeMB. Size for byte is useless cause the unit of region size in pd is MB.

**Comment by siddontang** [comment]
any update?

**Comment by huachaohuang** [comment]
Before merging this PR, please make sure that a region will not be merged in a short time after it has been split :)

**Comment by Connor1996** [comment]
@huachaohuang okay, I am working on it.

**Comment by siddontang** [comment]
any update @Connor1996

**Comment by disksing** [comment]
Blocked by #1048

File: conf/config.toml [hunk 2]
```diff
@@ -17,6 +17,8 @@ initial-cluster-state = "new"
 lease = 3
 tso-save-interval = "3s"
 
+namespace-classifier = "table"
```
Comment (huachaohuang) [inline conf/config.toml]: The default value in `config.go` is still "default", should we update that?

**Comment by nolouch** [comment]
PTAL @huachaohuang @disksing

**Pull request merged**